R/epinowcast.R

```diff
@@ -46,6 +47,26 @@
 #' @return A object of the class "epinowcast" which inherits from
 #' [enw_preprocess_data()] and `data.table`, and combines the input data,
 #' priors, and output from the sampler specified in `enw_fit_opts()`.
+#' @details
+#' ## Priors
+#'
+#' The `priors` data.frame should have the following columns: `variable`,
+#' `mean`, and `sd`. The `variable` column expects the following values:
+#'  - refp_mean_int: Log mean intercept for parametric reference date delay
```

Comment:
Sorry I think I had slightly misunderstood the plan here. I think this should either be phrased as an example or should be documented in the module where the priors are produced. (as there are more modules than enw_refernece that produce priors). 

In addition, it is not that priors needs all of these things - anything not specified here will be drawn from the defaults in the individual modules.

this is making me think that what we might actually want is here a short example vignette that showcases setting the priors in a model?

Reply:
> Sorry I think I had slightly misunderstood the plan here. I think this should either be phrased as an example or should be documented in the module where the priors are produced. (as there are more modules than enw_refernece that produce priors).

Which module is this? Sorry, I'm not familiar with the whole code base. If we're to go by adding it to the example, how would you suggest it's done?

> In addition, it is not that priors needs all of these things - anything not specified here will be drawn from the defaults in the individual modules.

Yes, I probably didn't word it right but I meant that if you want to specify a custom prior, it has to have those expected values/names.


> this is making me think that what we might actually want is here a short example vignette that showcases setting the priors in a model?

Yes, it can also just be a section in the Getting Started vignette. Originally, it just wasn't clear to me how to specify my own prior.

Reply:
> this is making me think that what we might actually want is here a short example vignette that showcases setting the priors in a model?

I've now converted what I was thinking into a vignette. Do check if it's what you imagine. I didn't go as far as specifying an `epinowcast()` function because I thought it might be unnecessary.